extensions/integration-tests/src/test/schemaCompare.test.ts

```diff
@@ -270,6 +274,231 @@ suite('Schema compare integration test suite @DacFx@', () => {
 		const excludeResult3 = await schemaCompareService.schemaCompareIncludeExcludeNode(operationId, t2Difference, false, azdata.TaskExecutionMode.execute);
 		assertIncludeExcludeResult(excludeResult3, true, 0, 0);
 	});
+
+	test('Cancel schema compare dacpac to dacpac comparison', async function () {
+		this.timeout(5 * 60 * 1000);
```

Comment:
can you move this to ta beforeEach() so that it doesn't get missed if more tests are added in the future?

Reply:
I tried this, but seems like some mocha package issue- beforeEach is throwing a reference error. 
Keeping it as-is for now.

Reply:
The fact that you apparently don't have the mocha package locally concerns me. You should (and I've verified on my side that things seem to work as expected with adding the beforeEach in that file)

Did you do the full clean + rebuild? (Clean being stash everything -> git clean -fxd -> rebuild)

Reply:
I tried the full clean + rebuild. So I do have the mocha folder under node_modules, but it didn't have the mocha folder under ..\extensions\integration-tests\node_modules\@types\mocha. index.t.ts was missing as well. 
But after your change, it exists, but still beforeEach isn't working as expected.